src/pages/apps/index.jsx

```diff
@@ -0,0 +1,238 @@
+  const { apps, categories } = usePluginData('apps-plugin');
+
+  const sortedApps = apps
+    .sort((a, b) => {
+      if (activeSort === SORTS.ALPHABETICAL) {
+        return a.name.localeCompare(b.name);
+      } else if (activeSort === SORTS.RECENT) {
+        return new Date(a.date) > new Date(b.date) ? -1 : 1;
+      }
+    })
+    .filter((app) => {
+      if (activeQuery && activeQuery !== '') {
+        console.log(activeQuery, app.name.toLowerCase());
+        return app.name.toLowerCase().includes(activeQuery);
+      } else {
+        return true;
+      }
+    });
+
+  const filters = Object.entries(categories);
+
+  const renderSort = (sort) => {
+    return (
+      <li
+        key={sort}
+        className={clsx(
+          'dropdown__link',
+          activeSort === sort && 'dropdown__link--active',
+          styles.sortDropdownItem
+        )}
+        onClick={() => setActiveSort(sort)}
+      >
+        <span>{sort}</span>
+      </li>
+    );
+  };
+
+  const renderPillFilter = (filter) => {
+    const [name, entries] = filter;
+    return (
+      <li
+        key={name}
+        className={clsx(
+          'pills__item',
+          styles.pillFilter,
+          activeCategory && activeCategory[0] === name && 'pills__item--active'
+        )}
+        onClick={() => setActiveCategory(filter)}
+      >
+        <span>{name}</span>
+        <span className={clsx('badge badge--secondary', styles.filterBadge)}>
+          {entries.length}
+        </span>
+      </li>
+    );
+  };
+
+  const renderDropdownFilter = (filter) => {
+    const [name, entries] = filter;
+    return (
+      <li
+        key={name}
+        className={clsx(
+          'dropdown__link',
+          styles.dropdownFilterItem,
+          Array.isArray(activeCategory) &&
+            activeCategory[0] === name &&
+            'dropdown__link--active'
+        )}
+        onClick={() => setActiveCategory(filter)}
+      >
+        <span>{name}</span>
+        <span className={clsx('badge badge--secondary', styles.filterBadge)}>
+          {entries.length}
+        </span>
+      </li>
+    );
+  };
+
+  const currentFavs = activeCategory
+    ? activeCategory[1].filter((app) => app.isFavorite)
+    : sortedApps.filter((app) => app.isFavorite);
+
+  return (
+    <Layout title="App Showcase">
+      <main className="margin-vert--xl">
+        <h1 className={styles.title}>Showcase</h1>
+        <p className={styles.subtitle}>
+          Discover <strong>hundreds of production applications</strong> built
+          with Electron.
+        </p>
+        <div className={clsx('margin-bottom--md', styles.filtersContainer)}>
+          <div
+            className={clsx(
+              'container',
+              'margin-bottom--sm',
+              styles.sortAndSearchContainer
+            )}
+          >
+            <div className="dropdown dropdown--hoverable">
+              <button className="button button--secondary">
```

Comment:
```suggestion
              <button
                className={clsx(
                  'button button--secondary',
                  styles.sortDropdownButton
                )}
              >
```

```scss
// File: src/pages/apps/apps.module.scss
.sortDropdownButton {
  color: var(--ifm-color-content) !important;
  background-color: var(--ifm-color-emphasis-200);
  border-color: var(--ifm-color-emphasis-200);
}
```

Before:

https://user-images.githubusercontent.com/8198408/186056443-214da4d7-bef1-43bc-9f18-7ef98cbddd6a.mp4

After:

https://user-images.githubusercontent.com/8198408/186056463-3b5a4b4e-be42-4a51-a416-a75b4fc0b120.mp4